tests/lang/core.pk

```diff
@@ -37,11 +37,20 @@ assert(r.first == 1)
 assert(r.last == 5)
 
 ## Math functions
-from math import PI, sin, cos, tan, abs
+from math import PI, sin, cos, tan, abs, sinh,
+  cosh, tanh, asin, acos, atan
 
 assert(sin(0) == 0)
 assert(sin(PI/2) == 1)
 
+assert(sinh(0) == 0)
+assert(cosh(0) == 1)
+assert(tanh(0) == 0)
+
+assert(asin(0) == 0)
+assert(acos(1) == 0)
+assert(atan(0) == 0)
```

Comment:
Remove these lines as they're suppressed by the below lines.

Reply:
Okay, sure!

Reply:
Please mark it resolved once you've actually done it, till then let it be unresolved.